src/components/more-from-blogs/more-from-blogs.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../card";
+
+import * as styles from './more-from-blogs.module.scss';
+
+const MoreFromBlogs = ({ data, refMoreFrom, moreFromAuthor }) => {
```

Comment:
Do not you change the name to another more generic one?

Reply:
Done